definitions/procedures/packages/update.rb

```diff
@@ -27,7 +27,11 @@ def necessary?
     end
 
     def description
-      "Update package(s) #{@packages.join(', ')}"
+      if @yum_options.include?('--downloadonly')
```

Comment:
It surprises me that there's a `yum_options` parameter to the generic `Update` procedure. My immediate question is whether it also applies those on DNF.

Reply:
> It surprises me that there's a `yum_options` parameter to the generic `Update` procedure. My immediate question is whether it also applies those on DNF.

It does, as the DNF package manager inherits the yum one. I tested this on a real system.